rocketc/rocketc.py

```diff
@@ -290,7 +309,29 @@ def _update_user(self, user_id, username, email):
         if email != self.email:
             url_path = "users.update"
             data = {"userId": user_id, "data": {"email": email}}
-            response = self.request_rocket_chat("post", url_path, data)
+            response = self._request_rocket_chat("post", url_path, data)
             if response["success"]:
                 self.email = email
         self._set_avatar(username)
+
+    @XBlock.json_handler
+    def set_default_channel(self, data, suffix=""):
```

Comment:
Not an expert on xblocks, my question is when this method is called? what menas the decorator @XBlock.json_handler?

Reply:
This method is called by js when the person push a button and the decorator means that the method receives a json in data